app/app.go

```diff
@@ -656,7 +657,6 @@ func NewQuicksilver(
 		gov.NewAppModule(appCodec, app.GovKeeper, app.AccountKeeper, app.BankKeeper),
 		slashing.NewAppModule(appCodec, app.SlashingKeeper, app.AccountKeeper, app.BankKeeper, app.StakingKeeper),
 		distr.NewAppModule(appCodec, app.DistrKeeper, app.AccountKeeper, app.BankKeeper, app.StakingKeeper),
-		mint.NewAppModule(appCodec, app.MintKeeper, app.AccountKeeper, app.BankKeeper),
```

Comment:
what's the rationale for moving this down? what is the impact?

Reply:
ah - because it is now under Quicksilver modules, not sdk modules?

Reply:
Yeah, this doesn't impact functionality.  

I just moved mint to the  quicksilver modules for consistency.